group1/comm/static/comm_tool.js

```diff
@@ -680,3 +718,10 @@ function deleteMessage(msgid) {
     return false;
   }
 }
+
+// EVENT LISTENERS
+
+$("#startVideo").click(()=>{
```

Comment:
Missing space between ")" and "=".  Missing space between ">" and "{".

Reply:
resolved